vladiate/vlad.py

```diff
@@ -24,6 +25,8 @@ def __init__(
         self.delimiter = delimiter or getattr(self, "delimiter", ",")
         self.line_count = 0
         self.ignore_missing_validators = ignore_missing_validators
+        if disable_console_log:
+            self.logger.disabled = True
```

Comment:
This can just be `self.logger.disabled = disable_console_log` with no `if` statement.